source/newbiefaq.md

```diff
@@ -1,32 +1,32 @@
 ---
-title: Apache Newcomer FAQs
+Title: Apache Newcomer FAQs
```

Comment:
Can this be changed to title again?
```suggestion
title: Apache Newcomer FAQs
```

While it may work (I tested it, it does work) it does not follow the documentation of Hugo. Next to that I think it's good to use the same style website wide (`title:`).

Example snippet copied from https://gohugo.io/templates/lists/#example-project-directory:
```yaml
---
title: My Go Journey
date: 2017-03-23
publishdate: 2017-03-24
---

I decided to start learning Go in March 2017.

Follow my journey through this new blog.
```